artiq/frontend/artiq_ddb_template.py

```diff
@@ -117,25 +116,64 @@ def process_dio(self, rtio_offset, peripheral, num_channels=8):
                             "module": "artiq.coredevice.edge_counter",
                             "class": "EdgeCounter",
                             "arguments": {{"channel": 0x{channel:06x}}},
-                        }}
-                        """,
+                        }}""",
```

Comment:
Changing the formatting here of the output device_db file. Can you post an example one using this PR to show how it affects the output formatting?

Reply:
Between every standalone peripheral there is one blank line, except for `TTL` and `EdgeCounter` devices which insert two blank lines. This change makes those two devices in line with all the others: adding only one blank line between peripherals. For other devices, the strings used for generating the device db also have the triple quote directly after the `}}` (without inserting an extra blank line).